### DISCUSSION TITLE
Saving an entire project from qmd.file back to rmd.file

### DISCUSSION THREAD

**Discussion by KennethGottfredsen** [quarto-dev/quarto-cli#2447] [Q&A]
Hi guys, 

I want to write simple documents and books in R Studio with Quarto. The main reason is, that I prefer the diverse language integration that Quarto offers, compared to R markdown. My main question is:

Is it possible to save all my files from an entire book projekt or some documents back in the rmd.file? 

I'm asking because I have to write an assignment. The requirements are that it has to be delivered in rmd.file format. My main concern is that I have to copy paste everything and get a ton of errors in doing so - and in the process violate make it impossible to reproduce my work. 

I hope you guys can help me out. 

/Kenneth

**Comment by mcanouil** [comment]
The only conversion tool in Quarto is from/to Jupyter Notebook.
If you manage to not use really specific features of Quarto, you simply have to update the metadata header (yaml header) to make it work properly with `knitr`/`rmarkdown`.

**Reply by KennethGottfredsen** [reply]
When you say update, do you mean adding a specific function that work with knitr/rmarkdown in the yaml header?

**Reply by mcanouil** [reply]
I meant, you update/replace the yaml header, e.g., `format: html` to `output: html_document`, _etc._

**Reply by KennethGottfredsen** [reply]
Ok thank you for the fast response.

**Reply by cderv** [reply]
> The requirements are that it has to be delivered in rmd.file format.

If your assignment is a Rmd file, I don't think using Quarto features is among the expected output. You could either ask if you can provide a `.qmd` file instead, or just stick with R Markdown for this project. 

Otherwise, what @mcanouil said here. 👍
